lib/std/build.zig

```diff
@@ -1149,6 +1149,7 @@ pub const LibExeObjStep = struct {
     name_prefix: []const u8,
     filter: ?[]const u8,
     single_threaded: bool,
+    code_model: ?builtin.CodeModel,
```

Comment:
I suggest:
```suggestion
    code_model: builtin.CodeModel = CodeModel.default,
```
and then the code below should omit the `-code-model` parameter when the value is default.